doc/Rules.md

```diff
@@ -1510,7 +1510,7 @@ Rationale: Violations of this rule can lead to improperly rendered content.
 
 <a name="md039"></a>
 
-## MD039 - Spaces inside link text
+## MD039 - Spaces inside the link text
```

Comment:
As mentioned in a previous comment, "text" here doesn't refer to a specific instance, so there's no need to add "the".